commands/proofs/tx.go

```diff
@@ -22,6 +22,10 @@ data to other peers as needed.
 }
 
 func doTxQuery(cmd *cobra.Command, args []string) error {
+	if err := commands.RequireInit(cmd); err != nil {
```

Comment:
not thrilled that we have to do this with every cmd func. Can we not move it into root command, so it only needs to be called in one place ? Or otherwise have some wrapper convention ? I'm worried new commands will neglect it ...